app/controllers/ctc/portal/dependents_controller.rb

```diff
@@ -18,7 +18,7 @@ def destroy
       action: 'removed',
       client: current_client
     )
-    if current_intake.dependents.filter(&:eligible_for_child_tax_credit_2020?).length.zero?
+    if current_intake.dependents.filter(&:eligible_for_child_tax_credit?).length.zero?
```

Comment:
i feel like this will need the tax year?

Also, it might make more sense to instantiate the eligibility classes in this file? 🤔